src/redux/missions/missionsSlice.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-param-reassign */
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const missionApiUrl = 'https://api.spacexdata.com/v3/missions';
+
+const initialState = {
+  missions: [],
+  isSucceed: false,
+  isLoading: false,
+  error: null,
+};
+
+const fetchMissions = createAsyncThunk(
+  'missions/fetchMissions',
+  async () => {
+    try {
+      const response = await fetch(missionApiUrl);
+      const data = await response.json();
+      return data;
```

Comment:
- [ ] Kindly filter the incoming data before returning it for the reasons discussed earlier since we don't need the entire information coming from the API. We just need a small piece of information for the application to work. ✍️

**Necessary pieces of information**
<img src="https://user-images.githubusercontent.com/17243325/225840310-600f035e-5206-44a0-a8fa-51916be02d1e.png" width="300">